content/spin/manifest-reference.md

```diff
@@ -119,13 +121,38 @@ Each table in the `component` array contains the following fields:
 |                         |            |             | If `type = "spin"` there are no other keys defined. In this case, Spin calls the component using a standard Wasm component interface. Components built using Spin SDKs or Spin interface files use this convention. | `{ type = "spin" }` |
 |                         |            |             | If `type = "wagi"`, Spin calls the component's "main" (`_start`) function using [a CGI-like interface](https://github.com/deislabs/wagi). Components built using languages or toolchains that do not support Wasm interfaces will need to be called in this way. In this case, the following additional keys may be set:<br/><br/>* `argv` (optional): The string representation of the `argv` list that should be passed into the handler. `${SCRIPT_NAME}` will be replaced with the script name, and `${ARGS}` will be replaced with the query parameters of the request, formatted as arguments. The default is to follow the CGI specification, and pass `${SCRIPT_NAME} ${ARGS}`<br/><br/>* `entrypoint` (optional): The name of the function to call as the entry point to this handler. By default, it is `_start` (which in most languages translates to `main` in the source code).<br/><br/>See [the HTTP trigger documentation](http-trigger) for details. | `{ type = "wagi" }` |
 
-### The `component.trigger` Table for Redis Applications
+### Additional Fields for `trigger.redis` Tables
 
 | Name                    | Required?  | Type        | Value    | Example   |
 |-------------------------|------------|-------------|----------|-----------|
 | `channel`               | Required   | String      | The Redis channel which this component handles. Messages on this channel will cause the component to execute. | `"purchases"` |
 
-### The `component.build` Table
+## The `component` Table
+
+The keys of the `component` table are user-defined, and are usually written as part of the table syntax e.g. `[component.my-component]`.  The value of each key is another table with the fields shown in the following table.
```

Comment:
> The keys of the `component` table are user-defined,

This threw me for a loop for a minute thinking that it was referring to the inner table keys and would probably be even more confusing to anyone less familiar with TOML terminology. Perhaps clarify something like:

"The keys of the `component` table (e.g. `my-component` in `[component.my-component]`)"

Reply:
Good catch - thank you.  I've updated as you suggest, and the resulting text is:

> The keys of the `component` table - such as the `my-component` in `[component.my-component]` - are user-defined, and are usually written as part of the table syntax e.g. `[component.my-component]`.

It's clear, but do you think it's okay that it repeats the "[component.my-component]" thing?  I can play around with the wording if not.

Reply:
More clear but yeah a bit awkward.

Also I just noticed:

> table with the fields shown in the following table.

which might cause table-madness. Maybe "table with the following fields:"

Reply:
Huh, that is indeed a lot of tables.  Thanks!